api/python/provisioner/config.py

```diff
@@ -64,6 +64,7 @@
 
 
 REPO_CANDIDATE_NAME = 'candidate'
+RELEASE_INFO = 'RELEASE.INFO'
```

Comment:
This is a file name and this const can be elaborated to avoid confusion with the ENUM below

Reply:
Fixed